### PR TITLE
CODEOWNERS

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Added the start of the CODEOWNERS file.

### CODE REVIEW

**Comment by FoxxoTrystan** [comment]
Explain this file please.

**Comment by dvir001** [comment]
> Explain this file please.

It auto assign this:
image

File: .github/CODEOWNERS.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Last match in file takes precedence.
+
+# Config files
+/Resources/ConfigPresets/* @Cheackraze
+
+# Mapchecker
+/github/mapchecker/* @TsjipTsjip
+/github/workflows/frontier-mapchecker.yml @TsjipTsjip
```
Comment (TsjipTsjip) [inline .github/CODEOWNERS.txt:8]: Correct paths:
`.github/mapchecker/*`
`.github/workflows/frontier-mapchecker.yml`

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]

**Comment by FoxxoTrystan** [comment]
When this is aproved i may add some of my shuttles/code on it.

**Comment by dvir001** [comment]
> When this is aproved i may add some of my shuttles/code on it.

You can just send me the edit your want on Discord so I can quick add it

**Comment by dvir001** [comment]
Note this file is based on #863 with changes under the upstream uncommitted lines for the sake of no merge conflicts.

**Comment by dvir001** [comment]
We can really start using this for ships, I made sure this file wont have conflicts issue with the one coming from upsteam merge.

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**